lib/drawers/publish-page.vue

```diff
@@ -179,6 +198,15 @@
       </form>
     </ui-collapsible>
 
+    <!-- manual title updating -->
+    <ui-collapsible class="publish-section publish-title" title="Page Title">
+      <form class="publish-title-form" @submit.prevent="saveTitle">
+        <span class="title-description">Manually update the page title that appears in the Clay Menu. Will be overwritten when saving components that set the title.</span>
```

Comment:
I don't like the wording of this (I dont think writers or editors know what does/doesn't update the title)

Reply:
The wording will be tweaked as we get feedback, but I agree